src/main.rs

```diff
@@ -174,7 +174,10 @@ enum Command {
 #[derive(StructOpt, Debug)]
 #[structopt(flatten)]
 pub struct NewOptions {
-    #[structopt(help = "name of the project")]
+    #[structopt(help = "location of the project root")]
+    pub project_root: String,
+
+    #[structopt(long, help = "name of the project", default_value = "")]
```

Comment:
Let's use an optional type to represent the missing value rather than an empty string.
```suggestion
    #[structopt(long, help = "name of the project")]
    pub name: Option<String>
```